templates/contactDetails.html

```diff
@@ -31,11 +31,11 @@
 		</header>
 		<section>
 			<div ng-repeat="prop in ctrl.contact.props | toArray | orderDetailItems:'$key'">
-				<detailsItem ng-repeat="propData in prop" name="prop.$key" data="propData" model="ctrl" index="$index" ng-class="[ 'details-item-' + prop.$key ]"></detailsItem>
+				<detailsItem ng-repeat="propData in prop" name="prop.$key" data="propData" model="ctrl" index="$index" ng-class="[ 'details-item-' + prop.$key, {'hidden': 'org title version fn geo photo uid'.indexOf(prop.$key) !== -1} ]"></detailsItem>
```

Comment:
I need help with this.
The best way is to access the detailsItem scope and check for ctrl.meta.template==='hidden'.
I can access it on the console through `.scope().$$childHead.ctrl.meta.template` But I can't in this template.

Any idea?
@DeepDiver1975 @Henni @owncloud/javascript